transitmatters/t-performance-dash
# Date Presets for the system page

## Motivation

Closes #655 
## Changes

1) rename datePresetConfig -> datePresetStore
2) set up presets for system section.


## Testing Instructions

Go to system > slow zones - see that the preset is automatically set.
Use the date presets and move around pages
